packages/react/error-boundary/src/ErrorBoundary.ko.md

```diff
@@ -0,0 +1,34 @@
+# ErrorBoundary
+
+- see https://jbee.io/react/error-declarative-handling-1/ 선언적으로 로딩과 에러 상태 처리하기
+- see https://toss.im/slash-21/sessions/3-1 Suspense와 에러 처리 관련된 Slash 21 발표
```

Comment:
Wouldn't it better to move them to the bottom of this documentation? Because they are not related with what `ErrorBoundary` is.

My suggestion is:
```markdown
# ErrorBounday

선언적으로 에러를 관리하기 위해서 사용하는 컴포넌트입니다.
`ErrorBoundary` 컴포넌트는 children의 render/useEffect에서 발생한 에러를 잡아 `renderFallback`으로 주어진 컴포넌트를 렌더링합니다.

...

## References
- [React에서 선언적으로 비동기 다루기 - jbee.io](https://jbee.io/react/error-declarative-handling-1/)
- [프론트엔드 웹 서비스에서 우아하게 비동기 처리하기 - SLASH 21](https://toss.im/slash-21/sessions/3-1)
```

Reply:
> they are not mainly related with what ErrorBoundary is.
I'm fully agree with you.. thanks for the new look👀

Reply:
[fix: reference position in doc](https://github.com/toss/slash/pull/102/commits/25a35b181b6e014f0060c5f2a8026aa3db0205b5)
[25a35b1](https://github.com/toss/slash/pull/102/commits/25a35b181b6e014f0060c5f2a8026aa3db0205b5)